dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/client/migration/MigrationInvoker.java

```diff
@@ -425,10 +428,10 @@ protected void refreshServiceDiscoveryInvoker(CountDownLatch latch) {
         }
         setListener(serviceDiscoveryInvoker, () -> {
             latch.countDown();
-            //TODO FrameworkStatusReporter
-//            FrameworkStatusReporter.reportConsumptionStatus(
-//                createConsumptionReport(consumerUrl.getServiceInterface(), consumerUrl.getVersion(), consumerUrl.getGroup(), "app")
-//            );
+            if (reportService.hasReporter()) {
+                reportService.reportMigrationStepStatus(
```

Comment:
change to `reportService.reportConsumptionStatus`

Reply:
done